pkgs/os-specific/linux/bluez/default.nix

```diff
@@ -44,6 +44,11 @@ in stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ] ++ lib.optional doCheck "test";
 
+  patches = [
+    # disconnect fix -> https://github.com/bluez/bluez/commit/28ddec8d6
+    ./28ddec8d6.patch
```

Comment:
Can we fetch https://github.com/bluez/bluez/commit/28ddec8d6b829e002fa268c07b71e4c564ba9e16.patch instead?

Reply:
I was actually looking into github docs on how to do it properly! Thanks.